src/starknet.rs

```diff
@@ -1630,3 +1627,37 @@ pub(crate) mod handler {
         }
     }
 }
+
+thread_local!(pub static SYSCALL_HANDLER_VTABLE: std::cell::Cell<*mut ()>  = const { std::cell::Cell::new(null_mut()) });
```

Comment:
There`s two space characters between the type and the assignment symbol (`=`).
Is the const block necessary? And the Cell too, I've seen that `LocalKey` has methods to mutate it but I can't seem to make an example work in the rust playground.

Reply:
Nice catch on the duplicated space. Probably `cargo fmt` didn't format it as it's inside a macro

The const block was suggested by clippy: `warning: initializer for thread_local value can be made const`

I think the cell is necessary, `replace` is only implemented for LocalKey<Cell<T>>: https://doc.rust-lang.org/std/thread/struct.LocalKey.html#impl-LocalKey%3CCell%3CT%3E%3E

Reply:
Done (removed the duplicated space)